subsys/net/l2/openthread/Kconfig.thread

```diff
@@ -88,7 +88,13 @@ config OPENTHREAD_CSL_SAMPLE_WINDOW
 
 config OPENTHREAD_CSL_RECEIVE_TIME_AHEAD
 	int "CSL receiver wake up margin in units of 10 symbols"
-	default 3
+	default 480
+
+config OPENTHREAD_CSL_MIN_RECEIVE_ON
+	int "Minimum CSL receive window"
+	default 5696
```

Comment:
Why such a weird value?

Reply:
It comes from here: https://github.com/openthread/openthread/blob/main/src/core/config/mac.h#L409-L420